solutions/swb-reference/README.md

```diff
@@ -68,11 +68,232 @@ This step is necessary to setup Service Catalog portfolio and products
+                            },
+                            "ParameterType": "String",
+                            "Description": "EC2 instance type to launch",
+                            "ParameterKey": "InstanceType"
+                        },
+                        {
+                            "IsNoEcho": false,
+                            "ParameterConstraints": {
+                                "AllowedValues": []
+                            },
+                            "ParameterType": "Number",
+                            "Description": "Number of idle minutes for auto stop to shutdown the instance (0 to disable auto-stop)",
+                            "ParameterKey": "AutoStopIdleTimeInMinutes"
+                        },
+                        {
+                            "IsNoEcho": false,
+                            "ParameterConstraints": {
+                                "AllowedValues": []
+                            },
+                            "ParameterType": "String",
+                            "Description": "The IAM policy to be associated with the launched workstation",
+                            "ParameterKey": "IamPolicyDocument"
+                        },
+                        {
+                            "DefaultValue": "1.1.1.1/1",
+                            "IsNoEcho": false,
+                            "ParameterConstraints": {
+                                "AllowedValues": []
+                            },
+                            "ParameterType": "String",
+                            "Description": "CIDR to restrict IPs that can access the environment",
+                            "ParameterKey": "CIDR"
+                        }
+                    ],
+                    "updatedAt": "2022-08-11T15:27:53.895Z",
+                    "sk": "ET#<id number>",
+                    "owner": "########-####-####-####-############",
+                    "description": "An Amazon SageMaker Jupyter Notebook",
+                    "id": "########-####-####-####-############",
+                    "pk": "ET#<id number>",
+                    "productId": "<product id>",
+                    "type": "sagemakerNotebook"
+                }
+            ]
+        }
+        ```
+        If there are no environment types displayed please follow instructions [here](../swb-reference/SETUP_v2p1.md###setup-project-configurations,-environmentType,-and-environmenttypeconfig) to create a new environment type
+
+    2. Choose the Environment Type that integration test will use as default when creating any Environment and copy the values from properties `id` and `type` from request.
+
+    3. In `./integration-tests/config` directory assign `id` value to `envTypeId` property and `type` value to `envType` property in `<STAGE>.yaml` file 
+
+
+3. For `envTypeConfigId` open Postman Collection and select `List envTypeConfigs` inside `envTypeConfig` folder.
```

Comment:
Are we not going to use all env types and all env type configs for tests? That would be the most thorough. We could have the integ tests do the list calls and use each permutation of env type and env type config that matches

Reply:
this is only the defaults for integration testing to create environments, if we extend later the integration tests to do all types then we will need then to add an array of all types to be tested, I am actually doing that in e2e testing

Reply:
got it. Yeah. let's make sure we are robust in our testing and extend the integ tests to be the same as the e2e in that sense. I will approve this readme section considering it is the _current_ state of the integ tests but we should add a backlog item to do all permutations of possible envs

Reply:
ticket created https://sim.amazon.com/issues/CHAM-179